server/journeys/record-investigation/routes.ts

```diff
@@ -23,11 +20,11 @@ function Routes(_csipApiService: CsipApiService): Router {
   return router
 }
 
-export default function routes(csipApiService: CsipApiService, prisonerSearchService: PrisonerSearchService): Router {
-  const router = Router({ mergeParams: true })
+export default function routes({ services, path }: { services: Services; path: string }) {
```

Comment:
I'm not sure I like this change - it goes from a function signature that specifies two services, to one that could specify any number in the types allowed - how is one supposed to know which to pass? Unless I've missed something?

Reply:
these routes are just passing along the services without making use of them itself, i don't think it's very useful to spell out each service when we can pass them in bulk.
the child routes that actually make use of the services can expand it to:
```
function Routes({ csipApiService }: Services)
```

that wasn't done in this particular file because there is no usage of csipApiService yet.

Reply:
Ah ok, makes sense now